### PR TITLE
Add AdonisJs

### PR DESCRIPTION
**Pull request by thetutlage**
Adonis is a true MVC framework for Nodejs with basics done right. It borrows the concept the of Service providers from popular PHP framework Laravel to write scalable applications and also leverage the power of ES6 to make your code expressive and maintainable.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for the suggestion, but I'm going to have to pass on this. It's hard for me to judge a framework [without](https://github.com/adonisjs/adonis-framework/graphs/contributors) much community. Happy to reconsider some time in the future when you've gathered more traction and community around the project ;)

File: readme.md [hunk 1]
```diff
@@ -298,6 +298,7 @@
 - [LoopBack](http://loopback.io) - Powerful framework for creating REST APIs and easily connecting to backend data sources.
 - [Meteor](https://www.meteor.com) - An ultra-simple, database-everywhere, data-on-the-wire, pure-Javascript web framework. *(You might like [awesome-meteor](https://github.com/Urigo/awesome-meteor))*
 - [SailsJS](http://sailsjs.org) - An MVC web framework with a modern twist, supporting WebSockets, streams, and a data-driven API.
```
Comment (knight444) [inline readme.md]: Hhhh

Comment (knight444) [inline readme.md]: 2

**Comment by thetutlage** [comment]
@sindresorhus Since the last time the project has gained lof of traction. Do u think it will be a good time to get it added to the list?

**Comment by RomainLanz** [comment]
ping @sindresorhus

**Comment by RomainLanz** [comment]
ping @sindresorhus...

**Comment by NitishTyagi** [comment]
👍

**Pull request merged**